docs/reference/feature-gates.md

```diff
@@ -0,0 +1,170 @@
+---
+title: Feature Gates
+---
+
+{% capture overview %}
+This page contains an overview of the various feature gates an administrator
+can specify on different Kubernetes components.
+{% endcapture %}
+
+{% capture body %}
+
+## Overview
+
+Feature gates are a set of key=value pairs that describe the features for alpha
+or experimental features.
+An administrator can use the `--feature-gates` command line flag on each component
+to turn a feature on or off.
+The following table is a summary of the feature gates that you can set on
+different Kubernetes components.
+
+- The "KCM" column and the "CCM" column are refering to the controller manager
+  component and the cloud controller manager component respectively.
+- The "Since" column contains the Kubernetes release when a feature is introduced
+  or its release stage is changed.
+- The "Till" column, if not empty, contains the last Kubernetes release in which
+  you can still use a feature gate.
+- The "Y" values in the cells indicate that a specific feature gate can be applied
+  to a specific component.
+
+| Feature | Default | Stage | APIServer | KCM | Kubelet | Scheduler | CCM | Proxy | Since | Till |
```

Comment:
I'm seeing an issue w/ the CSS where it's clipping off the right end of the table (see below screenshot). Consider moving the *Since* and *Till* columns to the left, after *Stage*, because I think it's more important for them to be seen. Also, consider renaming "Till" to "Until" so that it is more clear, especially to non-native English speakers.

<img width="908" alt="screen shot 2017-11-27 at 2 03 50 pm" src="https://user-images.githubusercontent.com/24798125/33292053-b900ddca-d37c-11e7-97a7-35227b9415e3.png">